integration-tests/wallet-no-annotations-call-by-index.spec.ts

```diff
@@ -3,13 +3,13 @@ import { noAnnotCode, noAnnotInit } from "./data/token_without_annotation";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract made with wallet API with no annotations calling methods by index using: ${rpc}`, () => {
+  describe(`Test contract origination with no annotations calling methods by index through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    test('Test contract made with wallet API with no annotations for methods', async (done) => {
+    test('Verify wallet.originate for a contract with no annotations for method', async (done) => {
```

Comment:
Here also, it is not the origination that matters, but the calls we make to the methods of the contrat that have no annotation, so we need to refer to them using indexes.

Reply:
changed to
"Verify calls to the methods of the contract that have no annotations and must be referred to using indexes."